examples/sdk/auth-delegated/mobile/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "mobile",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "ENVFILE=.env.android react-native run-android",
+    "ios": "ENVFILE=.env.ios react-native run-ios",
+    "lint": "eslint .",
+    "start": "react-native start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@dfns/sdk-react-native": "0.3.3-alpha.1",
```

Comment:
will update to a stable release. can't use relative path here within the repo. doesn't seem like there's a way to transpile typescript dependency with react native like how we were able to with reactjs and nextjs.